dat/missions/dvaered/dvaered-colonel-transport.lua

```diff
@@ -0,0 +1,56 @@
+--[[
+<?xml version='1.0' encoding='utf8'?>
+<mission name="Escort a Dvaered colonel to a system">
```

Comment:
I would recommend a more simple name. Something like `Dvaered Colonel Transport`. You can then start the mission from the console with `naev.missionStart("Dvaered Colonel Transport")` and it should spawn the NPC.